### PR TITLE
feat: allow usage of native docker-compose without specifying env

### PR DESCRIPTION
**Pull request by thepiwo**
closes https://github.com/aeternity/aeproject/issues/431

### CODE REVIEW

**Review by marc0olo — APPROVED** [review]
just tested, perfect :)

File: src/init/update-artifacts/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3.6'
 services:
 
   aeproject_node:
-    image: aeternity/aeternity:${NODE_TAG}-bundle
+    image: aeternity/aeternity:${NODE_TAG:-latest}-bundle
```
Comment (davidyuk) [inline src/init/update-artifacts/docker-compose.yml:5]: make sense to specify node version as well 🤷‍♀️

**Review by davidyuk — APPROVED** [review]
closes #431

File: src/init/update-artifacts/docker-compose.yml (hunk 1 above)
Comment (marc0olo) [reply src/init/update-artifacts/docker-compose.yml:5]: @davidyuk I agree generally but let's discuss tagging strategy with operations in that regards. I think we should have one major version tag like `6` or `7` which always points to the latest release of that major version. shortly discussed with @thepiwo but didn't ask @dincho or anybody else so far 😅

**Pull request merged**